docs/releases/Embeddinator-4000_0.2.md

```diff
@@ -0,0 +1,60 @@
+# Embeddinator-4000 0.2
+
+_work in progress_
+
+
+Requirements
+============
+
+* macOS 10.12 (Sierra) or later;
+* Xcode 8.3.2 or later;
+* Mono 5.0 or later;
+
+Optional
+--------
+
+* [Visual Studio for Mac](https://www.visualstudio.com/vs/visual-studio-mac/);
+* [Xamarin.iOS 10.11.0.136 or later](https://jenkins.mono-project.com/view/Xamarin.MaciOS/job/xamarin-macios-builds-master/) **Links to preview builds**. Required for iOS, tvOS and watchOS support;
+* [Xamarin.iOS 3.5.0.144 or later](https://jenkins.mono-project.com/view/Xamarin.MaciOS/job/xamarin-macios-builds-master/) **Links to preview builds**. Required for standalone macOS support;
+
+
+What's New
+==========
+
+This second release builds upon our initial version previewed at //build 2017.
+_to be completed_
+
+Objective-C Support
+-------------------
+
+* [pr284](https://github.com/mono/Embeddinator-4000/pull/284) - [interop] Added support for `IFormatProvider` as a Objective-C protocol;
+* [pr299](https://github.com/mono/Embeddinator-4000/pull/299) - [interop] Added support for managed types implementing `IEquatable<T>`;
+* [pr292](https://github.com/mono/Embeddinator-4000/pull/292) - [interop] `System.Decimal` is now mapped with Foundation's `NSDecimalNumber`
+* [pr315](https://github.com/mono/Embeddinator-4000/pull/315) - [generator] Better names are generated when types are used
+
+iOS Support
+-----------
+
+* [pr305](https://github.com/mono/Embeddinator-4000/pull/305) - [mtouch] Native symbols stripping now supported with Xamarin.iOS in embeddinator mode
+* [pr319](https://github.com/mono/Embeddinator-4000/pull/319) - [linker] Automatically create XML file to preserve anything required from SDK assemblies;
+
+macOS Support
+-------------
+
+* [pr324](https://github.com/mono/Embeddinator-4000/pull/324) - [mmp] Support for native frameworks generated with Xamarin.Mac;
+* [pr324](https://github.com/mono/Embeddinator-4000/pull/324) - [registrar] Support for subclassing managed subclasses of `NSObject` from Xamarin.Mac based assemblies;
```

Comment:
good point, the PR # and links can come later (before release) - it's the description that matters the most (and blaming the file will give the details)